public/scripts/repository/settings.js

```diff
@@ -1,10 +1,9 @@
 $(function() {
   const predefinedMessages = {
-    'mail_disabled_successful': "Mail service disabled for this repository",
```

Comment:
Are these messages no longer needed?

Reply:
No we are not showing them now. We have added a toggle for these events and we show a check mark beside the element when the change is successful